frontend/service/Services.ts

```diff
@@ -72,7 +72,7 @@ const createService = async ({
           (acc, [middlewareChain, config]) => {
             acc[middlewareChain] = {
               ...config,
-              rpc: CHAIN_CONFIG[asEvmChainId(MiddlewareChain.GNOSIS)].rpc,
+              rpc: CHAIN_CONFIG[chainId].rpc,
```

Comment:
i think `chainId` should come from `middlewareChain` here, `serviceTemplate` using `MiddlewareChain` keys

Reply:
in updateService below it's EvmChainId, and wherever we pass it it's evmChainId (from selectedAgentConfig.evmHomeChainId) - does that answer your consern?